meta-balena-common/recipes-containers/balena/balena/balena-healthcheck-image-load.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Load balena healthcheck image
+After=balena.service
```

Comment:
@acostach I think `Requires=balena.service` is missing here as well as the balena daemon needs to be up and running to load the healthcheck image.